Description: gmake to make
 Use make instead of gmake in the build system.
Author: Obey Arthur Liu <arthur@milliways.fr>
Last-Update: 2010-02-17
--- a/binsrc/sqldoc/vspx_doc.sh
+++ b/binsrc/sqldoc/vspx_doc.sh
@@ -41,7 +41,7 @@
 fi
 pwddir=`pwd`
 cd "${cutterdir}"
-gmake
+make
 cd "${pwddir}"
 
 begin_xml ()
